src/operations/extrusions/slice/isA.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Determin if the given object is a poly3.
+ * @params {poly3} object - the object to interogate
+ * @returns {true} if the object matches a poly3 based object
+ */
+const isA = (object) => {
```

Comment:
kudos for coherency, although I still really dislike the naming (as per our previous discussions) xxx.isA(object) is still very un-intuitive to read in code :D

Reply:
Hopefully, these will only be used internally, and never seen by users due to the very nice API.